src/ResourceManagement/MachineLearning/Microsoft.Azure.Management.MachineLearning/Management/WebService.cs

```diff
@@ -0,0 +1,273 @@
+
+
+            this.Definition = ws;
+
+            this.ResourceGroupName = resourceGroup;
+
+            this._client = client;
+        }
+
+        /// <summary>
+        /// Constructor for a WebService that takes a Microsoft.Azure.Management.MachineLearning.WebService for the internal Definition.
+        /// </summary>
+        /// <param name="definitionWebService"></param>
+        /// <param name="resourceGroup"></param>
+        /// <param name="client"></param>
+        public WebService(Microsoft.Azure.Management.MachineLearning.WebServices.Models.WebService definitionWebService, string resourceGroup, AzureMLWebServicesManagementClient client)
+        {
+            this.Definition = definitionWebService;
+            this._client = client;
+            this.ResourceGroupName = resourceGroup;
+        }
+
+        /// <summary>
+        /// Reads in a web service from a web service definition json file.
+        /// </summary>
+        /// <param name="webServiceParameters">Parameters to replace within the file if interpolation parameters exist.</param>
+        /// <param name="webServiceDefinitionFilePath">The path to the web service definition file.</param>
+        /// <returns></returns>
+        private Microsoft.Azure.Management.MachineLearning.WebServices.Models.WebService InputWebServiceFromWSDFile(Dictionary<string, string> webServiceParameters, string webServiceDefinitionFilePath)
+        {
+            string wsDefinition;
+
+            //StreamReader r = new StreamReader("file.txt"); System.IO.
+            using (StreamReader r = System.IO.File.OpenText(webServiceDefinitionFilePath))
+            {
+                wsDefinition = r.ReadToEnd();
+            }
+
+            foreach (var kvp in webServiceParameters)
+            {
+                wsDefinition = wsDefinition.Replace($"$({kvp.Key})", kvp.Value);
+            }
+
+            // TODO: call this in the constructor and assign 
+            return ModelsSerializationUtil.GetAzureMLWebServiceFromJsonDefinition(wsDefinition);
+        }
+
+        /// <summary>
+        /// Updates this web service and updates the internal state.
+        /// </summary>
+        /// <param name="otherWebService">The other web service from which this web service is being updated.</param>
+        public void Update()
+        {
+            Microsoft.Azure.Management.MachineLearning.WebServices.Models.WebService ws = this._client.WebServices.Patch(this.Definition, this.ResourceGroupName, this.Title);
+
+            this.Definition = ws;
+        }
+
+        /// <summary>
+        /// Updates this web service and updates the internal state. Takes an AutoRest web service model.
+        /// </summary>
+        /// <param name="otherWebService">The other AutoRest web service model from which this web service is being updated.</param>
+        internal void Update(Microsoft.Azure.Management.MachineLearning.WebServices.Models.WebService otherWebService)
```

Comment:
I think our goal is to avoid end users access classes under "Microsoft.Azure.Management.MachineLearning.WebServices.Models". I think here you want to have is "Microsoft.Azure.MachineLearning.WebService".